docs/index.md

```diff
@@ -409,68 +431,37 @@ If you want to add another cluster (like asia-east1 in the above screenshots), r
-    deploy-prod-$NEW_REGION:
-      # Only deploy when running on master in $APP-env
-      only:
-        refs:
-          - master
-      stage: deploy-prod-$NEW_REGION
-      image: gcr.io/cloud-builders/gke-deploy:stable
-      tags:
-      - cluster:prod-$NEW_REGION
-      script:
-        # Work around for https://github.com/kubernetes/kubernetes/issues/36072
-        # Stated solution is to use "--force" or "kubectl create"; gke-deploy does not have this option so the following works
-        # Get all services that have the "kubectl.kubernetes.io/last-applied-configuration" annotation and put into an environment variable
-        - export SERVICENAMES=$(kubectl get svc -o jsonpath="{range .items[?(@.metadata.annotations.kubectl\.kubernetes\.io/last-applied-configuration)]}{.metadata.name}{':'}{end}")
-        # for each service name, remove the annotation
-        - echo $SERVICENAMES | tr -d '\n' | tr ':' '\0' | xargs -0 -I {} kubectl annotate svc {} kubectl.kubernetes.io/last-applied-configuration-
-        - /gke-deploy run --filename prod.yaml
-    EOF
-    ```
-
-1. Now make similar changes in your `$APP_NAME-env` repo. This time we're going to make our changes in the `staging` branch, which already was created when we made our app above.
+1. Now add a new environment for the application using the cluster.
 
     ```shell
     export APP_NAME="go-app"
-    cd $APP_NAME-env
-    git checkout staging
-    git pull
-    cat >> .gitlab-ci.yml <<EOF
-    deploy-prod-$NEW_REGION:
-      tags:
-      - cluster:$SHORT_CLUSTERNAME, app:$APP_NAME
-    stages:
-    - deploy-$SHORT_CLUSTERNAME
-    EOF
+    export NEW_REGION="asia-east1"
+    export SHORT_CLUSTERNAME="prod-$NEW_REGION"
+    cd $APP_NAME
+    appctl env add $SHORT_CLUSTERNAME --cluster ${SHORT_CLUSTERNAME} --region ${NEW_REGSION} --namespace ${APP_NAME}
     ```
 
-1. Commit the changes:
+1. Push the changes:
 
     ```shell
-    git add .gitlab-ci.yml
-    git commit -m "adding new cluster to CD"
     git push
     ```
 
-1. Now in the Gitlab web UI, navigate to the link shown in the output of your last git command, which should look like:
+1. Prepare the release candidate for the new environment:
 
-    ```console
-    remote: To create a merge request for staging, visit:
-    remote:   https://$GITLAB_HOSTNAME/$APP_NAME/$APP_NAME-env/merge_requests/new?merge_request%5Bsource_branch%5D=staging`
+    ```shell
+    export SHORT_CLUSTERNAME="prod-$NEW_REGION"
+    appctl prepare ${SHORT_CLUSTERNAME}
     ```
 
-1. Choose "Submit merge request".
+1. Click the Merge Request link in the console output, you will see the change in the deployment repo for the new environment. Click "Merge".
 
-1. Click "Merge" (don't choose to delete the source branch).
+1. Apply the release candidate for the new environment:
 
-   Note that for $app-env, you will have to push to staging branch first, then perform an MR.
+    ```shell
+    export SHORT_CLUSTERNAME="prod-$NEW_REGION"
```

Comment:
`SHORT_CLUSTERNAME` has already been exported.

Reply:
reverted the change.